### ISSUE TITLE
Issue with `npx @grafana/toolkit plugin:create my-plugin`

### ISSUE THREAD

**Issue by Skeen** [grafana/grafana#28019]
**What happened**:

I attempted to run `npx @grafana/toolkit plugin:create my-plugin` inside `node:12`, but it reports:
```
npm ERR! Failed at the node-sass@4.14.1 install script.
```

**What you expected to happen**:

A folder structure would appear in `/my-plugin/`

**How to reproduce it (as minimally and precisely as possible)**:

```
> docker run -it node:12 /bin/bash
root@0aec4a38af65:/# npx @grafana/toolkit plugin:create my-plugin
```

**Anything else we need to know?**:

Full output from the command:
```
internal/modules/cjs/loader.js:968
  throw err;
  ^

Error: Cannot find module '/root/.npm/_npx/7/lib/node_modules/@grafana/toolkit/node_modules/node-sass/scripts/install.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-sass@4.14.1 install: `node scripts/install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-sass@4.14.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-10-03T13_59_24_665Z-debug.log
Install for [ '@grafana/toolkit@latest' ] failed with code 1
```

![peek](https://user-images.githubusercontent.com/225151/94993581-4e2f0300-0592-11eb-84db-a518057089e3.gif)

**Comment by ccm73** [comment]
I had the same issue; for me it was resolved when I ran `npx @grafana/toolkit plugin:create my-plugin` as a non-root user who had full write/create permissions to the directory structure where the plugin was being created.

**Comment by Betula-L** [comment]
**TLDR:** `npm config set ignore-scripts true` may help this issue.

I had encountered the same problem as follow when running `npx @grafana/toolkit plugin:create my-plugi`
```
Error: Cannot find module '/data/root/.npm/_npx/16849/lib/node_modules/@grafana/toolkit/node_modules/puppeteer-core/install.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
```

It is because that do not have access to mkdir directory when execute npm install scripts, however, that's not necessary for create plugin project. Just forbiden it, everything works.

**Comment by Skeen** [comment]
Seems to be working for me @Betula-L.

**Comment by shashwataNichesoft** [comment]
![Screenshot 2021-07-22 105643](https://user-images.githubusercontent.com/84380189/126594759-e7e5d1a1-edb8-47a4-bc7a-f8d8f391ea22.png)

executed `Yarn install` after pulling a working plugin from another machine
